functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+function legoCollection (array $legoSets) : string {
+
+    if($legoSets === []){
+        $displayLego = 'There is an error with the database, please try again later.';
+    } else {
+        $displayLego = '';
+        foreach ($legoSets as $set) {
+
+            if ($set['retired'] == 0) {
+                $buyLink = "<a href='" . $set['buy-URL'] . "' target='_new'>Buy " . $set['item-name'] . '</a>';
+            } else {
+                $buyLink = "<a class='retired'>Retired Product</a>";
+            }
+
+            if ($set['star-rating'] === '5') {
```

Comment:
could you do this with a for loop? just loop over the number of times and concatenate on 1 star per loop iteration